webapp-django/crashstats/monitoring/views.py

```diff
@@ -169,57 +169,20 @@ def healthcheck(request):
         es.info()  # will raise an error if there's a problem with the cluster
 
         # Check SuperSearch paginated results
-        assert_supersearch_counts()
+        assert_supersearch_no_errors()
 
     return {'ok': True}
 
 
-def assert_supersearch_counts():
-    """Make sure that all shards in ElasticSearch return all results.
-    If some nodes in the cluster are unhealthy, the whole
-    SuperSearch query might succeed but the number of records
-    might not match the "total" count provided in every result page.
-
-    This test has been tested in our production database, using the
-    /api/SuperSearch endpoint and the epsilon hovers around 0.005 roughly
-    and that's *with* webapp caching at play.
-    """
+def assert_supersearch_no_errors():
+    """Makre sure an uncached SuperSearch query doesn't have any errors"""
     supersearch = SuperSearch()
     # We don't want any caching this time
     supersearch.cache_seconds = 0
     results = supersearch.get(
         product=settings.DEFAULT_PRODUCT,
-        _results_number=0,
+        _results_number=1,
         _columns=['uuid'],
+        _facets_size=1,
     )
```

Comment:
So that it queries at least one. Or else ES might not be exercised enough. If it seems that you only need 0 rows, perhaps it won't start the query engine. 
And for `_facets_size` we don't need them. But setting it to 0 is like not setting it at all I vaguely remember.